Bubble Sort Algorithm

```diff
@@ -0,0 +1,37 @@
+import random
+
+num_elements = int(input("Enter the number of elements in the list: "))
```

Comment:
Interesting approach with size input but it supposes input validation.
Entering string/ or negative value/ or float provokes error